icons/droplet.json

```diff
@@ -8,7 +8,8 @@
   ],
   "tags": [
     "water",
-    "weather"
+    "weather",
+    "liquid"
```

Comment:
```suggestion
    "liquid",
    "fluid",
    "wet",
    "moisture",
    "damp",
    "bead",
    "globule"
```

Reply:
Can you please also add these to `droplets.json`?